client/js/test/table-view.test.js

```diff
@@ -28,6 +28,24 @@ describe('table-view', () => {
   });
   });	
 
+  describe('table-foot', () => {
+    it('table foot has the sum of col values', () => {
```

Comment:
Test assertions read like a sentence, so they should begin with a verb. This one would say, "It table foot has the sum of col values". A better test assertion would say "it DISPLAYS the sum of column values in the table footer".